src/main/java/ByeCommand.java

```diff
@@ -0,0 +1,10 @@
+/**
+ * Represents a "bye" command given to TaskList.
+ */
```

Comment:
Based on our course's coding style, I think the verb at the beginning (`represents`) should be `represent` instead. Likewise for other method documentations in your code.